eclass/cmake-utils.eclass

```diff
@@ -441,6 +441,8 @@ cmake-utils_src_prepare() {
 	fi
 
 	popd > /dev/null || die
+
+	CMAKE_UTILS_SRC_PREPARE_HAS_RUN=1
```

Comment:
I'd like to keep it semi-official for the reason that some packages call cmake in a subproject of some package. For such exceptional cases `CMAKE_UTILS_SRC_PREPARE_HAS_RUN` can be defined globally to disable the QA check.